### PR TITLE
Add feature getDiagnostics for cameras, and voltage diagnostics

### PR DESCRIPTION
**Pull request by oysteingr**

### CODE REVIEW

File: tests/components/diagnosticsMessage.spec.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import sinon from 'sinon';
+import chai, { expect } from 'chai';
+import sinonChai from 'sinon-chai';
+import { DiagnosticsMessage, MinMaxDiagnosticsMessage, DiagnosticsLevel } from './../../src/components/diagnosticsMessage';
+
+
+chai.should();
+chai.use(sinonChai);
+
+describe.only('DiagnosticsMessage', () => {
```
Comment (jontore) [inline tests/components/diagnosticsMessage.spec.ts]: Remove only.

File: src/components/diagnosticsMessage.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  get type(): string {
+    return this._type;
+  }
+
+  get level(): DiagnosticsLevel {
+    return this._level;
+  }
+
+  get tip(): string {
```
Comment (jontore) [inline src/components/diagnosticsMessage.ts]: How is tip supposed to be used?

Comment (oysteingr) [reply src/components/diagnosticsMessage.ts]: Good Q, missing impl. Added 'Check your cables' tip  to voltage diagnostic message

File: tests/components/diagnosticsMessage.spec.ts (hunk 1 above)
Comment (oysteingr) [reply tests/components/diagnosticsMessage.spec.ts]: 👍

**Pull request merged**